content/docs/command-reference/plots/diff.md

```diff
@@ -6,7 +6,8 @@ overlaying them in a single image. This allows to compare them easily.
 ## Synopsis
 
 ```usage
-usage: dvc plots diff [-h] [-q | -v] [--targets [<path> [<path> ...]]]
+usage: dvc plots diff [-h] [-q | -v]
+                      [--targets [<paths> [<paths> ...]]]
```

Comment:
- [ ] Corresponding change required in core (also check in `diff` and `params diff`)